hooks/msghooks.py

```diff
@@ -120,6 +120,11 @@ def receive_message():
                 return f"{reply} Here you go: 🍪"
             else:
                 return f"{reply} No."
+        if re.match(r"sudo (pl(s|z|ease) )?make meh? a coo?kie", without_ping):
+            if random.random() < 0.25 or not (message["user_id"] == 281362 or message["user_id"] == 497390 or message["user_id"] == 475866):
```

Comment:
Why does sudo only work 25% of the time? And why are these users not allowed to use it? Why not check if you're an RO and say "NO." otherwise?